app/controllers/issue_templates_controller.rb

```diff
@@ -41,23 +40,22 @@ def index
       end
     end
 
-    @globalIssueTemplates = GlobalIssueTemplate.joins(:projects).where(["projects.id = ?", @project.id]).order('position')
+    @global_issue_templates = GlobalIssueTemplate.joins(:projects).where(["projects.id = ?", @project.id]).order('position')

```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Line is too long. [124/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)